PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -169,6 +169,12 @@
     jetTable.variables.puId, doc="Pileup ID flags with 106X (2017) training"
 )
 
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
```

Comment:
I think it would be better to handle it as in #40485, namely to remove CSVv2 in the default config, and put it back for the `rune2_nanoAOD_any` modifier. Run3 should be considered the "default" configuration whenever possible, and changes for other eras should be made wrt that. @vlimant do you concur?

Reply:
addressed in [cc2047c](https://github.com/cms-sw/cmssw/pull/40539/commits/cc2047c89771ad2cfb6fdb3709ead4bae4601c0c)